Applications/Spire/Include/Spire/KeyBindings/KeyBindings.hpp

```diff
@@ -136,7 +139,7 @@ namespace Details {
         \param sequence The key sequence.
         \param action The action.
       */
-      void set_binding(QKeySequence sequence, Action action);
+      void set_binding(QKeySequence sequence, const Action& action);
```

Comment:
Changed to const reference here because the internal data structure does not consume rvalues.